lib/wrapbox/runner/ecs.rb

```diff
@@ -248,9 +268,13 @@ def fetch_task_status(cluster, task_arn)
       end
 
       def register_task_definition(container_definition_overrides)
```

Comment:
[nits] I think "register" is misleading a little. How about "prepare" or "build"?